src/modules/content-seeding/news-aggregator.js

```diff
@@ -97,12 +155,13 @@ function getDueFeeds (): Promise<Array<FeedsObject>> {
 
 function getRoomSpecificNews (): Promise<Array<RoomSpecificNews>> {
 	return performReadQuery({
-		$: `SELECT rooms.id as roomid, article.rawjson as rawjson, article.url as url FROM rooms 
+		$: `SELECT rooms.id as roomid, rooms.name as roomname, article.rawjson as rawjson, article.url as url FROM rooms 
 			JOIN LATERAL (
 				SELECT * FROM articles
-	   			WHERE articles.terms @@ plainto_tsquery(rooms.name)
+	   			WHERE articles.terms @@ plainto_tsquery(rooms.name) AND NOT rooms.tags @> &{noNewsRoom}
 	   			ORDER  BY articles.rawjson->>'published' DESC NULLS LAST LIMIT  1
-			) article ON TRUE`
+			) article ON TRUE`,
```

Comment:
SELECT rooms.id as roomid, rooms.name as roomname, article.rawjson as rawjson, article.url as url FROM 
(rooms 
 			JOIN LATERAL (
 				SELECT * FROM articles
	   			WHERE articles.terms @@ plainto_tsquery(rooms.name) AND NOT rooms.tags @> &{noNewsRoom}
 	   			ORDER  BY articles.rawjson->>'published' DESC NULLS LAST LIMIT  1
			) article ON TRUE
) roomsArticles INNER JOIN botnews WHERE NOT (botnews.room = roomsArticles.id )`,